### PR TITLE
Scorecount branch

### PR DESCRIPTION
**Pull request by 11maxed11**
- implemented ScoreKeeper class
- implemented TextDrawer class
- Changed sprites for the cannon and background

### CODE REVIEW

**Comment by iiKoe** [comment]
The Libgdx libraries you added should also be added to gradle. You could have seen the build fail in the commit log (red dot behind your commit when you view it on GitHub).

**Comment by iiKoe** [comment]
Also make sure you run "gradlew.bat" in the project root folder (CMD) check fix all errors.
Errors can be found in the reports in bustamove\core\build\reports\
checkstyle and pmd should have NO warnings or errors, findbugs reports a number of warnings which I still have to go over, so as long as findbugs only gives warnings its ok for me (but no errors)

File: core/src/com/group66/game/helpers/ScoreKeeper.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+	
+
+	// getting the current score
+	public int getCurrentScore() {
+		return currentScore;
+	}
+
+	//calculating the new score
+	public void setCurrentScore(int poppingBalls, int hangingBalls) {
+		this.currentScore = this.currentScore + (poppingBalls + (2^hangingBalls))* 10; //CHECK is this correct?
```
Comment (iiKoe) [inline core/src/com/group66/game/helpers/ScoreKeeper.java]: I think its is correct, the original game had a max score for 17 or more dropped balls, but lets just keep it like this for now, as there where also time based score bonuses. (so keep it and remove the CHECK comment)

"Popped" bubbles (that is, bubbles of the same color which disappear) are worth 10 points each. However, dropped bubbles (that is, bubbles that were hanging from popped bubbles), are worth far more: one dropped bubble scores 20 points; two score 40; three score 80. This figure continues doubling for each bubble dropped, up to 17 or more bubbles which scores 1,310,720 points" -Wikipedia https://en.wikipedia.org/wiki/Puzzle_Bobble#Scoring_system

File: core/src/com/group66/game/helpers/TextDrawer.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+		parameter.size = Config.FONT_SIZE; //font size
+		parameter.characters = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789.!'()?:"; //list of printable chars
+		myFont = generator.generateFont(parameter);
+		myFont.setColor(Color.WHITE);
+		generator.dispose();
+	}
+	
+	//function for drawing the player score
+	public void drawScore(SpriteBatch batch, int score){
+		myFont.draw(batch, "Score: " + Integer.toString(score), 430, 40);
```
Comment (iiKoe) [inline core/src/com/group66/game/helpers/TextDrawer.java]: Is the 430, 40 the location? If so, move to Config

File: core/src/com/group66/game/helpers/ScoreKeeper.java (hunk 1 above)
Comment (11maxed11) [inline core/src/com/group66/game/helpers/ScoreKeeper.java]: yes. this comment meant code-wise, as I'm not the world's best expert in Java.

File: core/src/com/group66/game/helpers/TextDrawer.java (hunk 2 above)
Comment (11maxed11) [inline core/src/com/group66/game/helpers/TextDrawer.java]: Moved.

**Comment by jeroenoverman** [comment]
Looks fine

**Pull request merged**